stable/mongodb/templates/headless-svc-rs.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.replicaSet.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "mongodb.fullname" . }}-headless
+  labels:
+    app: {{ template "mongodb.name" . }}
+    chart: {{ template "mongodb.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  type: ClusterIP
```

Comment:
Should this be configurable?

Reply:
nop, this one should be ClusterIp and ClusterIp: Nono